src/main/java/org/jabref/gui/fieldeditors/SimpleEditor.java

```diff
@@ -72,4 +72,8 @@ public Parent getNode() {
     public void requestFocus() {
         textInput.requestFocus();
     }
+
+    public void setEditable(boolean isEditable) {
+        textInput.setEditable(isEditable);
+    }
```

Comment:
Implement this method in "MarkdownEditor", because you cast on line 145 to it.

Reply:
Done！